application/run-full-batch.sh

```diff
@@ -2,13 +2,12 @@
 
 echo "{\"message\": \"Preparing to run ID Sync batch\", \"batch\": \"full\", \"idp_name\": \"${IDP_NAME}\", \"app_env\": \"${APP_ENV}\", \"id_store_adapter\": \"${ID_STORE_ADAPTER}\"}"
 
-output=$(/data/yii batch/full 2>&1)
+/data/yii batch/full 2>&1
 
 # If it failed, exit.
 rc=$?;
 if [[ $rc != 0 ]]; then
-  echo $output;
-  echo "{\"message\": \"FAILED: ID Sync batch. Exit code ${rc}. Message: ${output}\", \"batch\": \"full\", \"idp_name\": \"${IDP_NAME}\", \"app_env\": \"${APP_ENV}\", \"id_store_adapter\": \"${ID_STORE_ADAPTER}\"}"
+  echo "{\"message\": \"FAILED: ID Sync batch. Exit code ${rc}. \"batch\": \"full\", \"idp_name\": \"${IDP_NAME}\", \"app_env\": \"${APP_ENV}\", \"id_store_adapter\": \"${ID_STORE_ADAPTER}\"}"
```

Comment:
I think some bit of syntax was accidentally broken here.
```suggestion
  echo "{\"message\": \"FAILED: ID Sync batch. Exit code ${rc}.\", \"batch\": \"full\", \"idp_name\": \"${IDP_NAME}\", \"app_env\": \"${APP_ENV}\", \"id_store_adapter\": \"${ID_STORE_ADAPTER}\"}"
```